packages/perseus-editor/src/widgets/__stories__/numeric-input-editor.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+
+import NumericInputEditor from "../numeric-input-editor";
+
+type StoryArgs = Record<any, any>;
+
+type Story = {
+    title: string;
+};
+
+export default {
+    title: "Perseus/Editor/Widgets/NumericInput Editor",
+} as Story;
+
+export const Default = (args: StoryArgs): React.ReactElement => {
+    return <NumericInputEditor onChange={() => {}} />;
```

Comment:
One thing I might do is make this `onChange` callback an `action("onChange")` so that if you make changes in the editor you'll see the onChange callback data in the Storybook UI.